openhub/management/commands/import_affiliated_committers_data.py

```diff
@@ -0,0 +1,13 @@
+from openhub.data import get_data
+from django.core.management.base import BaseCommand
+from openhub.affiliated_committers import import_data
+
+
+class Command(BaseCommand):
+    help = 'Import AffiliatorCommitters data'
+
+    affiliated_committers = 'affiliated_committers'
```

Comment:
use a CONSTANT name, common in all of these Command, so that these Command `handle` method is identical.

You could, and should, create a `BaseCommand` which they all inherit `handle` from.

Reply:
>You could, and should, create a BaseCommand which they all inherit handle from.

@jayvdb, not getting it properly. Can you explain it a little bit more?

Reply:
CONSTANTS should be in CAPS.

Create class CONSTANTs for the two things which are different between `import_affiliated_committers_data.py`, `import_portfolio_projects_data.py`, etc.

Dont worry about the `BaseCommand` bit for now.

Reply:
DO NOT create class called `CONSTANTS`.  Read what reviewers say carefully.

Each of these `Command` classes should have only four members, 'help', 'DATA_NAME' and 'IMPORT_FUNC' and `handle`, and the `handle` must be _identical_ in all of them.